docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,442 @@
+---
+title: "Make an LED Blink With Buttons And With Code"
+linkTitle: "Blink an LED"
+weight: 30
+type: "docs"
+description: "Use the Viam app and the SDKs to make an LED turn on and off with a Raspberry Pi."
+webmSrc: "/img/blink.webm"
+mp4Src: "/img/blink.mp4"
+videoAlt: "A blinking L.E.D. connected to a Raspberry Pi"
+images: ["/img/blink.gif"]
+aliases:
+    - /tutorials/make-an-led-blink-with-the-viam-app/
+    - /tutorials/make-an-led-blink-with-a-raspberry-pi-and-sdk/
+    - /tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk/
+    - /tutorials/get-started/make-an-led-blink-with-the-viam-app/
+tags: ["board", "raspberry pi", "app", "sdk"]
+authors: []
+languages: [ "python", "go" ]
+viamresources: [ "board" ]
+level: "Beginner"
+date: "4 October 2022"
+cost: 70
+no_list: true
+---
+
+This tutorial will show you how to use Viam to make an LED blink with a Raspberry Pi.
+This is a great place to start if you have never built a robot or a circuit before.
+
+{{<gif webm_src="/tutorials/img/blink-an-led/image9.webm" mp4_src="/tutorials/img/blink-an-led/image9.mp4" alt="A GIF of the completed project showing a blinking blue LED connected to a Raspberry Pi with jumper cables." max-width="300px">}}
+
+First, you'll use the control interface on the [Viam app](https://app.viam.com) to turn the LED on and off.
+Then, you'll write code to control the LED using the Viam [software development kits](../../../program/).
+
+## What you'll need for this guide
+
+You will use the following hardware for this tutorial:
+
+- [Raspberry Pi 3 or 4](https://a.co/d/5Tn67G3)
+  - Refer to the [Viam Raspberry Pi Setup Guide](/installation/prepare/rpi-setup/) to set up your Pi.
+- [A solderless breadboard](https://www.amazon.com/dp/B0135IQ0ZC)
+- [Jumper wires](https://amzn.to/2qVhd4y)
+- [Resistor pack](https://amzn.to/2Dmainw)
+  - You will use a 100 Ohm resistor, which is the resistor with brown-black-brown bands
+- [An LED](https://amzn.to/2Ex2v5q)
+
+## Project setup
+
+Before you build your circuit, you need to set up the operating system on your Raspberry Pi and install `viam-server` on the Pi:
+
+1. Follow the [Raspberry Pi Setup Guide](/installation/prepare/rpi-setup/) to set up the operating system.
+2. Then [install `viam-server` and connect your Pi to the Viam app](/installation/#install-viam-server).
+
+If you encounter issues, reach out to us on the [Viam Community Discord](https://discord.com/invite/viam).
+
+## Build the circuit
+
+The next step is to build a simple LED circuit consisting of an LED and a resistor.
+The resistor protects the LED by limiting the flow of electricity through the circuit and is called a *current-limiting resistor*.
+
+You will make the LED controllable from the Raspberry Pi by connecting it to the [pins](https://en.wikipedia.org/wiki/Pin_(electronics)) on the Raspberry Pi.
```

Comment:
This sentence is still a bit awkward ( the will make controllable) - can you rephrase?
```suggestion
```